src/lib/create-app.ts

```
@@ -16,18 +15,35 @@ import { redis } from "./redis";
+        redis,
+        limiter: Ratelimit.slidingWindow(10, "10 s"),
+        ephemeralCache: cache,
+      });
+      this.instance = ratelimit;
+      return this.instance;
+    }
+    else {
+      return this.instance;
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)